.gitignore

```diff
@@ -4,7 +4,11 @@ __pycache__/
 *$py.class
 
 # tests
+logs/
 test_logs/
+reports/
+test_data/
+performance_assessment.py
```

Comment:
I don't think adding test_data/ and performance_assessment.py into the gitignore was intended (nor desirable).